src/topp/PercolatorAdapter.cpp

```diff
@@ -1114,12 +1127,15 @@ class PercolatorAdapter :
           }
         }
       }
-      //OPENMS_LOG_INFO << "No suitable PeptideIdentification for " << c_debug << " out of " << all_peptide_ids.size() << endl;
-      OPENMS_LOG_INFO << "Suitable PeptideHits for " << cnt << " found." << endl;
+      OPENMS_LOG_INFO << "Suitable PeptideHits for " << cnt << " of " << all_peptide_ids.size() <<  " PSMs found." << endl;
 
       // TODO: There should only be 1 ProteinIdentification element in this vector, no need for a for loop
       for (vector<ProteinIdentification>::iterator it = all_protein_ids.begin(); it != all_protein_ids.end(); ++it)
-      {      
+      {
+        // it is not a real search engine but we set it so that we know that
+        // scores were postprocessed
+        it->setSearchEngine("Percolator");
+        it->setSearchEngineVersion("3.02");
```

Comment:
is it possible to get this from percolator? otherwise we would need to document in the wiki that this needs to be updated if percolator is updated

Reply:
possible, yes. we have the same problem with many other search engines though. not sure if I want to do it right now.

Reply:
add Unknown?